Documentation/Books/AQL/Views/ArangoSearch/README.md

```diff
@@ -275,47 +468,94 @@ to match documents where 'description' best matches 'a quick brown fox'
     FOR doc IN someView SEARCH ANALYZER(doc.description IN TOKENS('a quick brown fox', 'text_en'), 'text_en')
       RETURN doc
 
-ArangoSearch sort
+ArangoSearch `SORT()`
 -----------------
 
-A major feature of ArangoSearch views is their capability of sorting results
+A major feature of ArangoSearch Views is their capability of sorting results
 based on the creation-time search conditions and zero or more sorting functions.
-The sorting functions are meant to be user-defined.
+The ArangoSearch sorting functions available are `TFIDF()` and `BM25()`.
 
-Note: Similar to other sorting functions on regular collections the first
-  argument to any sorting function is _always_ either the document emitted by
-  the `FOR` statement, or some sub-attribute of it. 
+Note: The first argument to any ArangoSearch sorting function is _always_ the
+document emitted by a `FOR` operation over an ArangoSearch View.
 
-The sorting functions are meant to be user-defined. The following functions are already built in:
+Note: An ArangoSearch sorting function is _only_ allowed as an argument to a
+`SORT` operation. But they can be mixed with other arguments to `SORT`.
 
-### Literal sorting
-You can sort documents by simply specifying the *attribute-name* directly, as you do using indices in other places.
+So the following examples are valid:
 
-### Best Matching 25 Algorithm
+```js
+FOR doc IN someView
+    SORT TFIDF(doc)
+```
+
+```js
+FOR a IN viewA
+    FOR b IN viewB
+        SORT BM25(a), TFIDF(b)
+```
+
+```js
+FOR a IN viewA
+    FOR c IN someCollection
+        FOR b IN viewB
+            SORT TFIDF(b), c.name, BM25(a)
+```
+
+while these will _not_ work:
+
+```js
+FOR doc IN someCollection
+    SORT TFIDF(doc) // !!! Error
+```
+```js
+FOR doc IN someView
+    RETURN BM25(doc) // !!! Error
+```
+```js
+FOR doc IN someView
+    SORT BM25(doc.someAttr) // !!! Error
+```
+```js
+FOR doc IN someView
+    SORT TFIDF("someString") // !!! Error
+```
+```js
+FOR doc IN someView
+    SORT BM25({some: obj}) // !!! Error
+```
+
+The following sorting methods are available:
+
+### Literal sorting
+You can sort documents by simply specifying arbitrary values or expressions, as
+you do in other places.
 
-`BM25(attribute-name, [k, [b]])`
+### BM25()
 
-Sorts documents using the [**Best Matching 25** algorithm](https://en.wikipedia.org/wiki/Okapi_BM25).
+`BM25(doc, k, b)`
 
-Optionally the term frequency **k** and coefficient **b** of the algorithm can be specified as floating point numbers:
+- *doc* (document): must be emitted by `FOR doc IN someView`
+- *k* (number, _optional_): term frequency, the default is _1.2_
```

Comment:
k and b are free parameters typically using for advanced optimization. There is nothing related to "term frequency"

Reply:
Fixed.